packages/snap/test/integration/general.test.ts

```diff
@@ -2,8 +2,8 @@ import { expect, use } from "chai";
 import sinonChai from "sinon-chai";
 import chaiAsPromised from "chai-as-promised";
 import { Dappeteer, DappeteerPage, initSnapEnv } from "@chainsafe/dappeteer";
-import { Methods } from "../../src";
 import { buildSnap } from "./utils";
+import { Methods } from "../../../types";
```

Comment:
please use package import, do not leave the workspace package to access some code on another package